src/steps/relay.py

```diff
@@ -34,6 +40,27 @@ def setup_main_relay_nodes(self, request):
         self.node2.start(relay="true", discv5_bootstrap_node=self.enr_uri)
         self.main_nodes.extend([self.node1, self.node2])
 
+    @pytest.fixture(scope="function")
+    def register_rln_main_relay_nodes(self, request):
+        logger.debug(f"Registering RLN credentials: {inspect.currentframe().f_code.co_name}")
+        rln_creds_source = "./rln_creds.json"
+        self.node1 = WakuNode(NODE_2, f"node1_{request.cls.test_id}")
+        self.node1.register_rln(rln_creds_source=rln_creds_source, rln_creds_id="1")
```

Comment:
don't we need a pytest.skip if the NODE_1 or NODE_2 is not nwaku?

Reply:
Fixed at https://github.com/waku-org/waku-interop-tests/pull/16/commits/932707cb016ee8c82509ea2783e445f2abbcc2e0

Reply:
Swapped at https://github.com/waku-org/waku-interop-tests/pull/16/commits/8840c80fa83afe4607e2ec379ae0c4c5a6cc3d8d